src/MemberProfile.js

```diff
@@ -1,17 +1,115 @@
 import React from "react";
 import { Link } from "react-router-dom";
+import MuiThemeProvider from 'material-ui/styles/MuiThemeProvider';
+import Chip from 'material-ui/Chip';
+import FontIcon from 'material-ui/FontIcon';
+import AppBar from 'material-ui/AppBar';
+import SvgIconFace from 'material-ui/svg-icons/action/face';
 
 const MemberProfile = ({ user }) => {
   return (
-    <div>
-      <img src={user.picture} alt="avatar" />
-      <h1>{user.name}</h1>
-      <p>{user.description}</p>
-      <p>{user.email}</p>
-      <p>GitHub: {user.githubName}</p>
-      <p>Flowdock: {user.flowdockName}</p>
-      <Link to="/me/edit">edit</Link>
+  <MuiThemeProvider>
+    <div className="container">
+      <div className="card">
+        <div className="pic">
+          <div className="container-nested">
+            <img src="http://placehold.it/225x225"/>
```

Comment:
That's pretty nifty. Didn't know that was a thing

Reply:
This is the user's profile picture right? We can use the actual user picture data pulled from the account you sign up with:
```js
<img src={user.picture} alt="avatar" />
```